admin-ui/plugins/auth-server/components/Clients/ClientUMADetailPage.js

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useState } from 'react'
+import Box from '@material-ui/core/Box'
+import { Link } from 'react-router-dom'
+import { FormControlLabel, Radio, RadioGroup } from '@material-ui/core'
+import moment from 'moment'
+import GluuLoader from 'Routes/Apps/Gluu/GluuLoader'
+import { Col, Container, FormGroup, Card } from 'Components'
+import { useParams } from 'react-router-dom'
+import { connect } from 'react-redux'
+import { getUMAResourcesByClient } from 'Plugins/auth-server/redux/actions/OIDCActions'
+import { getScope } from 'Plugins/auth-server/redux/actions/ScopeActions'
+import { getOidcDiscovery } from 'Redux/actions/OidcDiscoveryActions'
+import { useTranslation } from 'react-i18next'
+import isEmpty from 'lodash/isEmpty'
+import GluuLabel from 'Routes/Apps/Gluu/GluuLabel'
+import applicationStyle from 'Routes/Apps/Gluu/styles/applicationstyle'
+
+function ClientUMADetailPage({
+  clientData,
+  scope,
+  loading,
+  dispatch,
+  umaResources,
+}) {
+  const { t } = useTranslation()
+  const { id } = useParams()
+  const [selectedUMA, setSelectedUMA] = useState()
+  const [scopeExpression, setScopeExpression] = useState()
+
+  useEffect(() => {
+    if (isEmpty(umaResources)) {
+      dispatch(getUMAResourcesByClient(clientData?.inum))
```

Comment:
why again fetching UMA resources by client here? We can just pass it from `ClientCibaParUmaPanel.js`.

Reply:
done, updated @duttarnab